fapi-client/src/main/scala/com/gu/facia/api/utils/CardStyle.scala

```diff
@@ -20,7 +20,7 @@ object CardStyle {
   val news = "news"
 
   def apply(content: Content, trailMetaData: MetaDataCommonFields): CardStyle = {
-    val href = trailMetaData.href.orElse(content.safeFields.get("href"))
```

Comment:
Yes, `href` doesn't seem to ever appear in the `fields` of a CAPI item. I think this has probably come from things of the past, where we wanted an items `href` to fallback to the content `href` (of some sort).
But in this case it doesn't seem to matter as we only use it to check if it's an external link and style is accordingly.